src/panels/logbook/ha-logbook.js

```diff
@@ -51,7 +51,7 @@ class HaLogbook extends EventsMixin(PolymerElement) {
 
     <template is="dom-repeat" items="[[entries]]">
       <div class="horizontal layout entry">
-        <div class="time">[[_formatTime(item.when)]]</div>
+        <div class="date">[[_formatDateTime(item.when)]]</div>
```

Comment:
This makes for a very messy UI. Too much data that is not relevant. I think that we should never show a date if we're just showing a single day. We also should consider only showing a new date heading when the next entry is on another day than the previous one.